tailwind.config.cjs

```diff
@@ -34,33 +30,32 @@ module.exports = {
     },
     extend: {
       padding: {
-        4.5: "18px",
+        4.5: "18px"
       },
       width: {
-        2.5: "10px",
+        2.5: "10px"
       },
       height: {
         0.25: "1px",
-        50: "200px",
+        50: "200px"
       },
       borderWidth: {
         1: "1px",
-        1.5: "1.5px",
+        1.5: "1.5px"
       },
       gap: {
-        0.25: "1px",
+        0.25: "1px"
       },
       margin: {
-        0.25: "1px",
+        0.25: "1px"
       },
       boxShadow: {
-        sm1: "0px 2px 20px 0px rgba(0, 0, 0, 0.15)",
         md1: "0px 4px 16px 0px rgba(0, 0, 0, 0.12)",
-        md2: "0px 4px 20px 0px rgba(0, 0, 0, 0.15)",
+        md2: "0px 2px 20px 0px rgba(0, 0, 0, 0.15)",
```

Comment:
This merge conflict was resolved incorrectly, `sm1` should have `2px 20px` and `md2` should have `4px 20px`

Reply:
upon further analysis of the design I don't see 2px 20px used anywhere? on the style guide all cards and the group button seem to use 4px 20px. so I'll update md2 to be 4px 20px.